scripts/install/installer

```diff
@@ -557,37 +557,28 @@ class Node(AbstractConfig):
                     self._roles = node["roles"]
                     break
             else:
-                raise LookupError("Faild to find a node with name {}".format(self.name))
+                raise LookupError("Failed to find a node with name {}".format(self.name))
         return self._roles
 
 
 class JumpScale7(Node):
     def install_core(self):
-        jsversion = self.versions["jsversion"]
-        ovcversion = self.versions["ovcversion"]
-        self.remote.add_ssh_host("root", "git.gig.tech")
-        self.remote.add_ssh_host("root", "github.com")
-        self.remote.add_ssh_host("root", "docs.greenitglobe.com", 10022)
-
-        if (
-            self.remote.run("which python", check=False).exit_status != 0
-            or self.remote.run("which curl", check=False).exit_status != 0
-        ):
-            self.remote.run("apt-get upgrade -y && apt-get install -y python curl")
-        if self.remote.run("which js", check=False).exit_status != 0:
+        for image in self.versions["images"]:
+            if "openvcloud/base" in image:
+                break
+        fs_path = "/tmp/image_fs_{}.tar.gz".format(image)
+        if not self.first_node.exists(fs_path):
             cmd = """
-            set -e
-            export AYSBRANCH={0}; export JSBRANCH={0};
-            cd /tmp;rm -f install.sh;
-            curl -k https://raw.githubusercontent.com/jumpscale7/jumpscale_core7/{0}/install/install.sh > install.sh;bash install.sh
-            """.format(
-                jsversion.version
-            )
-            self.remote.run(cmd)
-        self.remote.write_file(
-            "/opt/jumpscale7/hrd/system/atyourservice.hrd",
-            AYSCONFIG.format(version=jsversion.version, ovcversion=ovcversion.version),
-        )
+            docker pull {0}
+            docker create --name install_base {0} true
+            docker export install_base > {1}
+            """.format(image, fs_path)
+            self.first_node.run(cmd)
+        buffer = BytesIO()
+        self.first_node.sftp.getfo(fs_path, buffer)
```

Comment:
this is a bad approach cause you will load a 1GiB image in memory (or however big it is)
i think on the remove we should be able to do
`self.remote.run('ssh <firstcontroler managementip> cat myimage | tar xf - -C /')`